cmd/helm/chart_pull.go

```diff
@@ -17,12 +17,18 @@ limitations under the License.
 package main
 
 import (
+	"fmt"
 	"io"
+	"path/filepath"
+
+	"helm.sh/helm/internal/experimental/registry"
 
 	"github.com/spf13/cobra"
 
 	"helm.sh/helm/cmd/helm/require"
+	"helm.sh/helm/internal/experimental/tuf"
```

Comment:
Now that v3 is using Go modules the major version needs to be in the import paths. 

helm.sh/helm/internal/experimental/tuf -> helm.sh/helm/v3/internal/experimental/tuf

Reply:
Thanks for the heads up, I'll rebase and update the import paths.